src/jdk.hotspot.agent/doc/transported_core.html

```diff
@@ -76,16 +76,35 @@ <h3>Using transported core dumps on Windows</h3>
  is setup:
  </p>
 
-<p>
+<code>
  set PATH=%JAVA_HOME%\bin;%JAVA_HOME%\bin\server;%PATH%
-</p>
+</code>
 
 <p>
 You can also include user JNI libraries in <b>PATH</b>.
 If the windows libraries are not identical, then they may also need to be copied
 to the debugger machine and included in <b>PATH</b>.
 </p>
 
+<p>
+By default symbols are also located using <b>PATH</b>. However, there are also Java properties that can be used to specify both the location of the binaries, and also separately the location of symbols. Use <b>sun.jvm.hotspot.debugger.windbg.imagePath</b> for the location of binaries, and use <b>sun.jvm.hotspot.debugger.windbg.symbolPath</b> for the location of symbols. <b>imagePath</b> defaults to <b>PATH</b> if not set, and <b>symbolPath</b> defaults to <b>imagePath</b>. The advantage of using these propeties is that you don't need to change your <b>PATH</b> setting, and they allow for binaries to be located separately from symbols.
```

Comment:
Is "also" needed here? The sentence makes more sense without it.

Reply:
The "also" is in reference to the previous paragraph mentioning that binaries are located using PATH. In other words, PATH is by default used to locate binaries and also to locate symbols.